lib/evaluateConstantSum.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+const clone = require('clone');
+
+const ConstantFraction = require('./ConstantFraction');
+const evaluateArithmetic = require('./evaluateArithmetic');
+const MathChangeTypes = require('./MathChangeTypes');
+const NodeCreator = require('./NodeCreator');
+const NodeStatus = require('./NodeStatus');
+const NodeType = require('./NodeType');
+
+// Evaluates a sum of constant numbers and integer fractions to a single
+// constant number or integer fraction. e.g. e.g. 2/3 + 5 + 5/2 => 49/6
+// Returns a NodeStatus object.
+function evaluateConstantSum(node) {
+  if (NodeType.isParenthesis(node)) {
+    node = node.content;
+  }
+  if (!NodeType.isOperator(node) || node.op !== '+') {
+    return NodeStatus.noChange(node);
+  }
+
+  // functions needed to evalute the sum
+  const summingFunctions = [
+    evaluateArithmetic,
+    ConstantFraction.addConstantFractions,
+    ConstantFraction.addConstantAndFraction,
```

Comment:
Does addConstantAndFraction add constants to variable fractions or to constant fractions? Or both. This suggests both.